src/react/deckgl.js

```diff
@@ -114,7 +128,11 @@ export default class DeckGL extends React.Component {
       cachePosition: false,
       centerOrigin: false,
       onClick: this._onClick,
-      onMouseMove: this._onMouseMove
+      onMouseMove: this._onMouseMove,
+      onDragStart: this._onDragEvent,
+      onDragMove: this._onDragEvent,
+      onDragEnd: this._onDragEvent,
+      onDragCancel: this._onDragCancel
```

Comment:
We're invoking picking on all of these events as long as one of them is specified. Can we add handlers only when they are not `noop`?